### DISCUSSION TITLE
port contol on unifi switches.

### DISCUSSION THREAD

**Discussion by JensjJ8200** [Kane610/aiounifi#674] [Ideas]
Hello
I hope, that this is the right place to make a sugestions in relation to https://www.home-assistant.io/integrations/unifi/

I would like to suggest to add port control feature. More specifc to be able to turn on and off a port via the Home Assistant integration. It is possible to do right now with PoE control. It's the enabled/disabled and restricted I am talking about. I know that it is possible to enable and diable a port via terminal / CLI.

**Comment by Kane610** [comment]
Thanks for the suggestion!

**Reply by JensjJ8200** [reply]
You are very velcome :-)

**Reply by Kane610** [reply]
Do you want to help out realizing it? :)

**Reply by JensjJ8200** [reply]
> Do you want to help out realizing it? :)

sure. I do however not know how to progam in that language. I know VBA 🤣. But how can I help?

**Reply by JensjJ8200** [reply]
https://community.ui.com/questions/We-would-like-to-control-the-ports-of-a-switch-via-ssh/5fd4c58e-1817-4fe6-9036-80437dadb036

Could this help?

**Reply by JensjJ8200** [reply]
Or how could I help?

**Reply by Gladbjerg** [reply]
@Kane610 
It seems like there is quite a few requests scattered on this feature, but it's in different threads. 

I don't know a lot about that kind of programming, but how can I/we help?
